features/step_definitions/machine_set.rb

```diff
@@ -5,4 +5,8 @@
   cb[cb_name] = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
 end
 
-
+Given /^I store the replicas of machineset named #{QUOTED} in the#{OPT_SYM} clipboard$/ do | machineset_name, cb_name |
```

Comment:
`/^I store the desired replicas of machineset named #{QUOTED} in the#{OPT_SYM} clipboard$/`

I think adding `desired` to name would make clear which replicas we are talking about.